### PR TITLE
add e621 support

### PR DESCRIPTION
**Pull request by AbbyAbbyAbbyAbby**
- Added `e621` as the downloader type
- Tested anchor exporting
- Tested file importing
- Tested single url and subscription
- Tested the test
- Specified the downloader in the test and anchor importer help
- Updated the readme
- __Added method in `importer.py` to parse e621 tags__

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -99,31 +99,32 @@ leave the test environment untouched so you can investigate the error.
...
+| fanbox                                  | no | yes | no |
+| webtoons                                | no | yes | yes |
+| kemono.party                            | no | yes | no |
+| seiso.party                             | no | yes | no |
+| baraag                                  | no | yes | yes |
+| pawoo                                   | no | yes | yes |
+| hentaifoundry                           | yes | yes | yes |
+| yande.re                                | yes | yes | yes |
+| rule34.xxx                              | yes | yes | yes |
+ | e621.net                                | yes | yes | yes |
```
Comment (thatfuckingbird) [inline README.md:127]: won't the indent break the table formatting?

**Review by thatfuckingbird — CHANGES_REQUESTED** [review]

File: hydownloader/importer.py [hunk 2]
```diff
@@ -72,6 +72,44 @@ def split_val(val):
...
+#         "general": [
+#             "furry",
+#             "gay_furry",
+#             "straight_furry"
+#         ]
+#     }
+# }
+# data should be the tags entry like "json_data['tags']
+def get_nested_tags_e621(data: dict[str, Any]) -> list[str]:
+    tags = list[str]()
```
Comment (thatfuckingbird) [inline hydownloader/importer.py:92]: though it works, according to python docs this is basically just the same as list(), see
https://docs.python.org/3/library/stdtypes.html#types-genericalias from the "The Python runtime does not..." paragraph.
If you want a type annotation that would be `tags: list[str] = []`

File: README.md (hunk 1 above)
Comment (AbbyAbbyAbbyAbby) [reply README.md]: It won't, but I didn't intend to make this change. Not sure what triggered it, didn't see it in PyCharm when pushing and it didn't happen with #22. I reverted the formatting changes.

File: hydownloader/importer.py (hunk 2 above)
Comment (AbbyAbbyAbbyAbby) [reply hydownloader/importer.py]: Formally typed language brain at work, suggested change made in new commit.

**Review by thatfuckingbird — APPROVED** [review]

**Pull request merged**